source/overview/faq.rst

```diff
@@ -265,29 +269,157 @@ Working for Mattermost
+
+Several of our customers value complete access to our source code and compile our plugins from scratch before deploying to their production servers. By adopting the Mattermost Source Available License, we can develop enterprise-only features in public without impacting this workflow.
+
+Will you distribute open-source plugin binaries without any licensing restrictions?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+At this time, we have no plans to distribute more than one version of each of our plugins. Without a Mattermost Enterprise E20 License, plugins may have reduced functionality or refuse to start altogether.
+
+Can I continue to use the existing open-source repositories without restriction?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes, the Mattermost Source Available License will only apply from the date it is added and to the versions in which it is included.
+
+Can I publish my own plugin under a license of my choosing?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Where can I find the license agreement for Mattermost Enterprise Edition?
+Yes, you are free to license your own code as you see fit. We will not apply our Source Available License either to the `starter-template <https://github.com/mattermost/mattermost-plugin-starter-template>`_ or `demo <https://github.com/mattermost/mattermost-plugin-demo>`_ plugins, leaving them under a permissive open source license to give you the freedom to develop your own plugins.
+
+Can I publish my own plugin and rely on enterprise specific functionality?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-See our `frequently asked questions about licensing <https://about.mattermost.com/pricing/#faq>`__.
+As before, you are free to license your own code as you see fit. Note that some server functionality is only enabled with a Mattermost Enterprise E20 license regardless of how you license your plugin.
+
+Can’t someone compile out any license restrictions?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We trust our community to honour the Mattermost Source Available License and work alongside us to develop features across our free and paid offerings. Our support team does not provide support to unlicensed, enterprise-only functionality.
+
+If I make my own plugin using your source available code, can I remove the license restriction?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No, the Mattermost Source Available License continues to apply to modifications.
+
+Can I write my own plugin and license it under the Mattermost Source Available License?
```

Comment:
I wonder if this is a better fit in an internal doc for our own team's use?

Reply:
Let's remove this section altogether as per discussion on community.